CONTRIBUTING.md

```diff
@@ -4,7 +4,42 @@
 
 Hi there! We're thrilled that you'd like to contribute to this project. Your help is essential for keeping it great. This project adheres to the [Open Code of Conduct][code-of-conduct]. By participating, you are expected to uphold this code.
 
-The majority of contributions won't need to touch any Ruby code at all.
+The majority of contributions won't need to touch any Ruby code at
+all.
+
+## Installation how to
+
+You might want to get Linguist running in order to test locally whatever
+changes you might be doing to it. You will need to install some
+prerrequisite libraries first. Run:
+
+	sudo apt-get install cmake
```

Comment:
I think it's Ok if you just say "install CMake", people will hopefully know how that works in their OS.